setup/file_permissions.rst

```diff
@@ -55,8 +55,7 @@ following script to determine your web server user and grant the needed permissi
 
 .. note::
 
-    The first ``setfacl`` command sets permissions on the existing files and
-    folders, while the second one sets permissions for future files and folders.
+    The first ``setfacl`` command sets permissions for future files and folder, while the second one sets permissions on the existing files and folders.
```

Comment:
If you add a line break after the first word crossing the 72th character, this PR would be perfect! :) (if you don't have the time, we can do it during merging).